js/ko/formatter.js

```diff
@@ -0,0 +1,11 @@
+import ko from "knockout";
+
+// make it global so it's accessible in views and give it a 'Knockout-y' name so it won't likely
+// be used in actual code
```

Comment:
Because I jokingly mentioned binding it to the ko global as an alternative: We're actually already doing that with `ko.modal`. So...?